zipkin-autoconfigure/metrics-prometheus/pom.xml

```diff
@@ -27,6 +27,7 @@
 
     <properties>
         <main.basedir>${project.basedir}/../..</main.basedir>
+        <prometheus_simpleclient.version>0.0.23</prometheus_simpleclient.version>
```

Comment:
Confirmed, Prometheus can succesfully scrape data with `0.0.26`. Pushed a commit; I'll start updating the Grafana dashboard for the new metric names, upload once this is merged, then we can hopefully also merge https://github.com/openzipkin/docker-zipkin/pull/135